product_category_tax/__manifest__.py

```diff
@@ -0,0 +1,13 @@
+# Copyright 2020 ForgeFlow S.L. (https://www.forgeflow.com)
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
```

Comment:
@AaronHForgeFlow @JordiBForgeFlow I will migrate this. Is the license AGPL or LGPL as they are mixed through the module ?

Reply:
I think AGPL is fine, as long as no other modules are built on top of this one.

Thank you.